fix: semantic tokens

reallybeard opened the pull request:
## Description

This updates most but not all of the uses for `useColorModeValue`.

We now use semantic tokens for example, if you want to do some text in an error color you would do `text.error`.

This also updates the theme with a darker/less blue version.

And finally, this removes our custom card component --- Chakra not has a built in Card component.

## Pull Request Type

- [ ] :bug: Bug fix (Non-breaking Change: Fixes an issue)
- [ ] :hammer_and_wrench: Chore (Non-breaking Change: Doc updates, pkg upgrades, typos, etc..)
- [x] :nail_care: New Feature (Breaking/Non-breaking Change)

## Issue (if applicable)

<!-----------------------------------------------------------------------------
If applicable, please link to the github issue and put `closes #XXXX` in your comment to auto-close the issue that your PR fixes.
------------------------------------------------------------------------------>

closes #

## Risk

<!-----------------------------------------------------------------------------
Outline the scope of your changes and the risk associated with them. You must use your discretion as an engineer to determine the potential impact of your changes.

E.g. an upgrade to `hdwallet` or core state management would be considered higher risk, and might require a full regression test. UI or isolated view changes, or something behind a feature flag may have near zero risk. Small bug fixes might require testing isolated to the specific fix.
------------------------------------------------------------------------------>
This is all theme related, shouldn't effect functionality

## Testing

<!-----------------------------------------------------------------------------
We treat every PR to be merged with the same scrutiny as if we were merging directly to production.

Your PR will not be merged if you do not complete the sections below for our engineering and operations teams to test.
------------------------------------------------------------------------------>

### Engineering

<!-----------------------------------------------------------------------------
Include sufficient information here for an engineer to test your PR. This may include how to test locally, in a built environment, changes to infrastructure etc.
------------------------------------------------------------------------------>

### Operations

<!-----------------------------------------------------------------------------
If your changes have a user-facing impact, describe how a non-technical QA team can functionally test your changes in a preview environment.

If they are not user-facing please describe how to test for any regressions that may occur.
------------------------------------------------------------------------------>
Nothing to test

## Screenshots (if applicable)
![Screenshot 2023-08-11 at 9 48 29 AM](https://github.com/shapeshift/web/assets/89934888/548e3473-0732-4053-8e48-927a925a4b1d)

0xApotheosis commented on src/components/TransactionHistory/TransactionsGroupByDate.tsx:
```diff
@@ -22,13 +22,12 @@ export const TransactionsGroupByDate: React.FC<TransactionsGroupByDateProps> = m
         const transaction = transactions[index]
         const transactionDate = dayjs(transaction.date * 1000)
-          .startOf('day')
+          .startOf('month')
```
This is a little weird now, as we just show the month and the time. Was this an intentional change?

<img width="1482" alt="Screenshot 2023-08-14 at 8 39 26 am" src="https://github.com/shapeshift/web/assets/97164662/2c8a8637-26bc-4157-a2f9-6b598141b2e6">

Note all of my scam NFTs 😕

reallybeard replied:
Yah I like grouping my month more than by day, but didn't update the format to include the full date on each transaction. But I can just leave this out for another future PR is not that big of a deal.

0xApotheosis reviewed (commented):
Phew. 

<img width="162" alt="Screenshot 2023-08-14 at 8 46 40 am" src="https://github.com/shapeshift/web/assets/97164662/d02bd598-b26a-4d75-8741-6aa966335fde">


Looking good. A few comments.

This is a bit gross:

<img width="500" alt="Screenshot 2023-08-14 at 8 47 59 am" src="https://github.com/shapeshift/web/assets/97164662/abd45fe2-bc3e-4d00-8ff4-485037c1e44c">

Icon too light?

<img width="603" alt="Screenshot 2023-08-14 at 8 49 01 am" src="https://github.com/shapeshift/web/assets/97164662/7b0ed246-2747-4018-99c6-1979530fb54d">

Search placeholder too light?

<img width="689" alt="Screenshot 2023-08-14 at 8 52 41 am" src="https://github.com/shapeshift/web/assets/97164662/201512cd-bf8a-4b82-bc47-15d523239b1f">

reallybeard commented:
Updated

![Screenshot 2023-08-13 at 11 54 41 PM](https://github.com/shapeshift/web/assets/89934888/5bc42ddc-d89c-4ed4-8128-247df55135aa)
![Screenshot 2023-08-13 at 11 54 45 PM](https://github.com/shapeshift/web/assets/89934888/7b1d1360-a59a-42e6-a4a4-d2125b9e5bef)

Put tx history back to grouping by month, and fixed the account dropdown on trade.

0xApotheosis reviewed (approved):
Solid.
